content/pages/education/apply.md

```diff
@@ -76,7 +74,7 @@ If you need to make a change (for example, you’re moving to a new school), man
 <div class="usa-alert usa-alert-warning usa-content va-alert" markdown="1">
 	<div class="usa-alert-body">
 
-##### Forms 22-1990N, 22-5490, and 22-5495
+##### Forms 22-1990N and 22-5495
 
 You must apply for education benefits using eBenefits if you're:
 - A dependent whose sponsor is permanently and totally disabled
```

Comment:
Question for @goldenmeanie: Now that the 5490 will be on Vets.gov, how should this language for the eBenefits redirect be amended? I wasn't sure if it was obvious to remove references to dependents here because the 5495 is for these kinds of dependents too, but only if they are managing an existing benefit. It seemed too wordy to say "A dependent whose sponsor is permanently and totally disabled and who is managing an existing benefit". What do you recommend?